orcid-web/src/main/java/org/orcid/frontend/web/controllers/WorksController.java

```diff
@@ -327,19 +327,6 @@ WorkForm getWorkInfo(@RequestParam(value = "workId") Long workId) {
                             ProfileEntity profileEntity = profileEntityCacheManager.retrieve(contributorOrcid);
                             String publicContributorCreditName = cacheManager.getPublicCreditName(profileEntity);
                             contributor.setCreditName(Text.valueOf(publicContributorCreditName));
-                            if(contributorOrcid.equals(getEffectiveUserOrcid())) {                                
```

Comment:
@bobcaprice @wjrsimpson This is going to be an issue. Researcher changes name to private but the record will still reflect the researchers name for every work (on their own record). Please research and discuss before tomorrow meeting.